src/FloorSubsystem/Floor.java

```diff
@@ -0,0 +1,56 @@
+package FloorSubsystem;
+import SchedulerSubsystem.Scheduler;
+import SchedulerSubsystem.SchedulerEvent;
+import Utlities.Passenger;
+import java.util.ArrayList;
+
+
+public class Floor implements Runnable {
```

Comment:
Implement a DestinationReceiver interface. (Destination event doesn't exist yet, this change can be done later by whoever does Floor).